src/run_atari_ppo.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+from mpi4py import MPI
+from baselines.common import set_global_seeds
+from baselines import bench
+import os.path as osp
+import gym, logging
+from baselines import logger
+
+def wrap_train(env):
+    from baselines.common.atari_wrappers import (wrap_deepmind, FrameStack)
+    env = wrap_deepmind(env, clip_rewards=True)
+    env = FrameStack(env, 4)
+    return env
+
+def train(env_id, num_frames, seed):
+    from baselines.ppo1 import pposgd_simple, cnn_policy
+    import baselines.common.tf_util as U
+    rank = MPI.COMM_WORLD.Get_rank()
+    sess = U.single_threaded_session()
+    sess.__enter__()
+    if rank != 0: logger.set_level(logger.DISABLED)
```

Comment:
Why disable the openai logger if rank != 0? That is what's keeping track of the useful metrics we want.

can you call logger.configure(os.path.join(args.logdir, rank))` instead, where we pass `args.logdir` from `main()`?